build_tools/python/e2e_test_framework/models/matmul.py

```diff
@@ -107,3 +107,43 @@
     entry_function="matmul_123x2561x2561_f32t_f32t_f32t_tile_config_default",
     input_types=["123x2561xf32", "2561x2561xf32", "123x2561xf32"],
 )
+
+MATMUL_1x256x2048_I8xI4_MLIR = common_definitions.Model(
+    id=unique_ids.MICRO_MATMUL_1x256x2048_I8xI4_MLIR,
+    name="matmul_1x256x2048_i8_i4_i32_tile_config_default",
+    tags=["i8i4", "ubench", "matmul"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_STABLEHLO_MLIR,
+    source_url="https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.25_1710887127/DOT_PRODUCT_JAX_1X256X2048XI8I4/stablehlo.mlirbc",
```

Comment:
I'm a little curious why we chose stablehlo? To me it seems to be easier to represent microbenchmarks in linalg? (and we can skip the import stage during compilation)

Reply:
I generated these from JAX and lowered to MLIR with StableHLO. Updated to use Linalg instead.